src/plugins/intel_npu/src/compiler_adapter/include/izero_adapter.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
Can't this be replaced by zero_adapter.hpp directly?

Reply:
will be, just waiting for the PR27095 to be merged. Much easier after this step.

Reply:
Unfortunately we still need to keep it. Just changed the name to ze_graph_ext_wrapper_interface.hpp. It is just a wrapper over templatized ze_graph_ext_wrapper object.